styles/index.css

```diff
@@ -0,0 +1,116 @@
+:root {
+    --main-bg-color: rgb(16, 3, 46);
+    --starwars-yellow: rgb(88%, 69%, 0%);
+    --logo-width: calc(150px * 1);
+    --nav-bar_height: 8em;
+}
+
+/* BACKGROUND */
+body {
+    background-color: var(--main-bg-color);
+    margin: 0;
+}
+
+.main-bg-image {
+    width: 100%;
+    height: 170%;
+    position: absolute;
+    background-position: center;
+    background-image: url("../images/background_stars.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    /* filter: grayscale(55%); */
+    backdrop-filter: var(--main-bg-color);
+    left: 0;
+}
+
+#main-containter {
+    display: grid;
+    align-items: center;
+    justify-items: center;
+    grid-template-columns: var(--logo-width) 1fr 1fr;
+    grid-template-rows: auto 1fr;
+    grid-template-areas:
+        "logo     bar     bar"
+        "content content content";
+}
+
+.sw-logo {
+    grid-area: logo;
+    align-self: start;
+    background-size: 100px;
+    width: 100px;
+    height: 50px;
+    background-image: url("../images/starwars_logo.png");
+    background-repeat: no-repeat;
+    margin-top: 1em;
+}
+
+/* QUIZ MENU */
+.quiz-main-menu {
+    margin-top: 1.5em;
+    box-sizing: border-box;
+    grid-area: bar;
+    display: flex;
+    margin-right: 1em;
+    max-height: a;
+    border: solid var(--starwars-yellow) 0.5rem;
+    border-radius: 3em;
+    background: rgba(0, 0, 0, 0.40);
+    box-shadow: 1px 1px 120px var(--starwars-yellow);
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+/* QUIZ MENU CONTENT */
+button {
```

Comment:
This could probably use some specific class as in the current form it will likely interfere with other buttons in the project. There will quite likely be some common styles shared by all buttons so creating the general `.btn` class and distributing it to the existing buttons is reasonable but at least for now the buttons it the header could have some only some class specific to them.

Reply:
Buttons could also use `cursor: pointer`.